src/game.h

```diff
@@ -471,6 +471,10 @@ class game
 
         item_location inv_map_splice( item_filter filter, const std::string &title, int radius = 0,
                                       const std::string &none_message = "" );
+        /**
+          * Execute customized inventory menu for the specified @ref gunmod.
```

Comment:
Presuming there is going to be more of these you could add doxygen family documentation:
```
/**
 * @name Item selection interface
 *
 * Your documentation here describing how the functions work
 */
/*@{*/

/** Very brief description only, fetches gunmods... */
item_location inv_for_gunmod( ...)


/** Very brief description only, fetches foo.. */
item_location inv_for_foo( ...)

/*@}*/
```